test/extended/testdata/oauth_idp/oauth-htpasswd.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+    - htpasswd:
+        fileData:
+          name: htpasswd
+      mappingMethod: claim
+      name: htpasswd
+      type: HTPasswd
+  templates:
```

Comment:
Why set templates and all the below if you're only trying to configure htpasswd?

Reply:
I need them there, now, since I'm using `oc replace` instead of patch